src/layouts/ApiLayout.res

```diff
@@ -1,5 +1,5 @@
 // This is used for the version dropdown in the api layouts
-let allApiVersions = [("latest", "v8.2.0"), ("v8.0.0", "< v8.2.0")]
+let allApiVersions = [("latest", "v11.0.0"), ("v10.0.0", "v8.2 - v10.1"), ("v8.0.0", "v6.0 - v8.2")]
```

Comment:
Just noticed that there is also a `v9.0.0`directory:
https://github.com/rescript-association/rescript-lang.org/tree/master/pages/docs/manual/v9.0.0

What about that one, should it also be added?

Reply:
Hmm, I think it should, the contents of the api subfolder of v9.0.0 is not identical with either v8.0.0 or v10.0.0.

Adding v9.0.0 would make the menu under "API" identical to the one under "Docs":

![Bildschirmfoto 2024-01-07 um 09 29 03](https://github.com/rescript-association/rescript-lang.org/assets/591384/f9671ff9-2ea8-4f0a-9cfb-da7e7e40dfc5)

Reply:
Done 70d8c2f7